src/components/Main/Scenario/ScenarioCardPopulation.tsx

```diff
@@ -21,12 +22,57 @@ countryOptions.push({ value: CUSTOM_COUNTRY_NAME, label: i18next.t(CUSTOM_COUNTR
 const caseCountOptions = caseCountsNames.map((country) => ({ value: country, label: country }))
 caseCountOptions.push({ value: NONE_COUNTRY_NAME, label: i18next.t(NONE_COUNTRY_NAME) })
 
+// According to the following link: https://github.com/i18next/react-i18next/issues/387,
+// i18next doesn't properly render colons if this isn't included
+i18next.init({
+  // allow keys to be phrases having `:`, `.`
+  nsSeparator: false,
+  keySeparator: false,
+})
+
 export interface ScenarioCardPopulationProps {
   errors?: FormikErrors<FormikValues>
   touched?: FormikTouched<FormikValues>
+  srcHospitalBeds?: string
+  srcICUBeds?: string
+  srcPopulation?: string
+  scenarioName: string
 }
-
-function ScenarioCardPopulation({ errors, touched }: ScenarioCardPopulationProps) {
+function getSrcStrings(srcHospitalBeds?: string, srcICUBeds?: string, srcPopulation?: string) {
+  let ret = ['Source: ', 'Source: ', 'Source ']
+  if (srcPopulation === undefined || srcPopulation === 'None') {
+    ret[0] = 'Source cannot be provided'
```

Comment:
Why are the negative messages not all the same (provided vs found?) I would prefer something like 'No data source in our records.' or similar for all of them.